src/api/entities/Asset/Compliance/Requirements.ts

```diff
@@ -1,6 +1,10 @@
 import { QueryableStorageEntry } from '@polkadot/api/types';
 import { Vec } from '@polkadot/types/codec';
-import { AssetCompliance, AssetComplianceResult, TrustedIssuer } from 'polymesh-types/types';
+import {
+  PolymeshPrimitivesComplianceManagerAssetCompliance,
+  PolymeshPrimitivesConditionTrustedIssuer,
+} from '@polkadot/types/lookup';
+import { AssetComplianceResult } from 'polymesh-types/types';
```

Comment:
After some experimenting it seems like we should be able to take out the types, but we'll need to figure out CountryCode generate section since that needs types.

We'll also need to go through all the createType calls and make sure none are using the old types either.

Reply:
I added in a script that deals with `Authorization` and that seems to work.

The `Result` ones seem to live in `src/polkadot/polymesh/types.ts`. I will take a look in the morning to see if they can get similar treatment.

Reply:
Seems like the script isn't good after all. The types and tests are happy, but when I tried it out it seems like the snake case type is returned (undefined errors). Using snake case for the field resolves it, so for now I am going to switch back to having another method for handling RPC authorization stuff.

Reply:
The country code stuff should be able to be generated from the metadata. I'll create a task for that